libs/wingsdk/API.md

```diff
@@ -3513,6 +3513,7 @@ The index of the element in the Json Array to return.
 | <code><a href="#@winglang/sdk.std.Json.keys">keys</a></code> | Returns the keys from the Json object. |
 | <code><a href="#@winglang/sdk.std.Json.parse">parse</a></code> | Parse a string into a Json. |
 | <code><a href="#@winglang/sdk.std.Json.stringify">stringify</a></code> | Formats Json as string. |
+| <code><a href="#@winglang/sdk.std.Json.tryParse">try_parse</a></code> | Try to parse a string into a Json. |
```

Comment:
Hey - just a small comment regarding auto generating the ref docs:
Please also update https://github.com/winglang/wing/blob/main/docs/04-reference/wingsdk-api.md
You don't need to make any of these doc updates manually. 
To generate these docs automatically after you made changes to the SDK code, please just run `npm run build` from the [wingsdk root folder](https://github.com/winglang/wing/tree/main/libs/wingsdk).

Reply:
OK.

But this is generated doc, committed by the mutate job, why is the other doc not covered by the mutate job?

Reply:
Good question, I'll check

Reply:
Here we have `add .` and the working directory is `libs/wingsdk` but the doc is in `../../docs`.

https://github.com/winglang/wing/blob/7e8d309d1be79c35c35bde3fb33f468eb18aa31a/.github/workflows/wingsdk-mutation.yml#L64

I think it should be `git add -u`

Reply:
#2219